gen/definitions/authorization_profile.yaml

```diff
@@ -60,17 +60,24 @@ attributes:
     type: Bool
     description: This attribute is mandatory when `web_redirection_type` value is `CentralizedWebAuth`. For all other `web_redirection_type` values the field must be ignored.
     example: true
+  - model_name: agentlessPosture
+    data_path: [AuthorizationProfile]
+    type: Bool
+    description: Agentless Posture.
+    example: false
   - model_name: accessType
     data_path: [AuthorizationProfile]
     type: String
     enum_values: [ACCESS_ACCEPT, ACCESS_REJECT]
-    description: Access type
+    description: "Allowed Values: `ACCESS_ACCEPT`, `ACCESS_REJECT`"
```

Comment:
We don't need to document allowed values in the description. If "enum_values" is included, this will be documented accordingly.

Reply:
ok got you. I fix that and also i changed description in new trust_sec definitions i added.